pkg/apis/discovery/validation/validation.go

```diff
@@ -147,3 +156,15 @@ func validatePorts(endpointPorts []discovery.EndpointPort, fldPath *field.Path)
 
 	return allErrs
 }
+
+func validateAddressType(addressType *discovery.AddressType, validAddressTypes sets.String) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	if addressType == nil {
```

Comment:
if this is not a pointer, testing for "" is OK

Reply:
Done, thanks!